meilisearch/tests/index/delete_index.rs

```diff
@@ -5,19 +5,19 @@ use crate::json;
 async fn create_and_delete_index() {
     let server = Server::new().await;
```

Comment:
Since we're not modifying the state of the server in the test we could use a shared index:
```rust
let server = Server::new_shared().await;
let index = server.unique_index();
```